app/graphql/types/record_type.rb

```diff
@@ -158,8 +164,8 @@ class RecordType < Types::BaseObject
     field :call_numbers, [String], null: true, description: 'Identification number used to classify and locate item'
     field :citation, String, null: true, description: 'Citation for item'
     field :edition, String, null: true, description: 'Edition information for item'
-    field :imprint, [String], null: true, deprecation_reason: 'Use `publicationInformation`'
-    field :publication_information, [String], description: 'Imprint information for item'
+    field :imprint, [String], null: true, deprecation_reason: 'Use `publishers`'
+    field :publication_information, [String], deprecation_reason: 'Use `publishers`'
```

Comment:
We should probably do a deprecation method here to populate `publication_information` from a concatenation of the various `publishers` subfields if `publication_information` does not exist. If these words make no sense, let me know and I can better describe what this might look like to allow for the smoothest transition for any consumers of this field.

Reply:
Oh, good call. I've not implemented something like that, but it does make sense. I'm going to check with Graham first to see whether there's data in all subfields before proceeding any further.

Reply:
Take a look at the methods that follow the RecordType definitions to see how this works. If the method name matches the field, it'll run instead of auto mapping.

So in this case, you'll create a method named `@object['publication_information']` and have it return whatever makes sense. Probably just pass through `publication_information` if it exists (until we fully rip that out of the data which may be soonish?) and construct a string of various concatenated sub fields from `@object['publication']` if not. Or something like that.

Reply:
Ha you pushed code as I was making that comment. FWIW I think it's probably fine. I don't think `publication_information` is long for this world (I think it goes away after full harvests for each source) so we likely don't need the complexity I suggested.

Reply:
I was just about to ask. :) In that case, I'll re-request review.